### ISSUE TITLE
[Accessibility] Narrator is not announcing position when focus is on expanded elements in "Multi-select uncontrolled" dropdown

### ISSUE THREAD

**Issue by SyedRehmanRazvi** [OfficeDev/office-ui-fabric-react#10903]
**User Experience:**
Users who rely on screen readers are getting affected as narrator is not announcing position of the expanded elements

**Repro Steps:**
Step 1: Start Narrator
Step 2: Open Fabric UI URL in Edge browser.
Step 3: Navigate to "Multi-select uncontrolled" dropdown available in "Dropdown" page and activate it
Step 4: Verify whether narrator is announcing position (Ex: 1 of 5) when navigating into expanded dropdown elements.

**Actual:**
Narrator is not announcing position when focus is on expanded elements in "Multi-select uncontrolled" dropdown
It is announcing as “Apple checkbox checked/Not checked"

**Expected:**
Narrator should announce position when focus is on expanded elements in Multi-select uncontrolled" dropdown
It should announce as “Apple checkbox checked/Not checked 1 of 4”

![image](https://user-images.githubusercontent.com/53210862/67090087-d15f1980-f1c6-11e9-965c-f3f39517a2c3.png)

**Comment by marygans** [comment]
@SyedRehmanRazvi thank you for submitting this issue!

I agree that Narrator should announce position when focus is on expanded elements in Multi-select uncontrolled" dropdown It should announce as “Apple checkbox checked/Not checked 1 of 4” the same way the controlled example does. 

I have repro's it and will investigate. 

Thanks!

**Comment by xugao** [comment]
with windows narrator, i cannot repro this anymore in Edge. but i can in chrome

**Comment by xugao** [comment]
@micahgodbolt  has fixed this bug here: https://github.com/OfficeDev/office-ui-fabric-react/pull/11374
